SCF-Session4-Question1/src/meta/test/FailTestCasesAllClasses.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * This test class is for testing all fail test cases
+ * @author ABHISHEK BISHNOI
+ * @version 1.0
+ * @since 17-July-2019
+ */
+package meta.test;
+
+import static org.junit.Assert.*;
+import meta.utility.ArrOperations;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FailTestCasesAllClasses {
```

Comment:
We don't create a separate class for FailTestCases. We can add both positive and negative test cases into a single test case file.